src/libraries/System.Formats.Cbor/tests/Reader/CborReaderTests.Tag.cs

```diff
@@ -192,6 +195,31 @@ public static void ReadDateTimeOffset_SingleValue_HappyPath(string expectedValue
             Assert.Equal(expectedValue.Offset, result.Offset);
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
```suggestion
        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Explain why this test cannot run on .NET Framework")]
```

Reply:
It's because of the remote executor, which has been giving me trouble with netfx in certain CI legs. At this point I just disable it from the start. I'll remove the suppression to see what happens.

Reply:
That's fine - just noting that the attribute has a `reason` parameter that I think we should populate if we decide to keep the attribute.